src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -564,6 +564,7 @@ export function getAppointmentSlots(startDate, endDate, forceFetch = false) {
 
     const startDateMonth = moment(startDate).format('YYYY-MM');
     const endDateMonth = moment(endDate).format('YYYY-MM');
+    const featureVAOSServiceRequests = selectFeatureVAOSServiceRequests(state);
```

Comment:
I think we should be using `selectFeatureVAOSServiceVAAppointments` here

Reply:
Makes sense. This is a direct booking and not an appointment request.